admin/includes/footer.php

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
For the footer, this footer will only be accessed by site administrators so the links in the footer should reflect links that may be useful to those making updates or changes to the site using a console. We probably don't need social media links, but think about what information would be useful if you're using a console.

We may also not need footer links but for sure keep the footer include so that we can include the closing body and html tags on all pages.

Reply:
I removed the icons and left basic links